v3/src/components/graph/plots/bar-chart/bar-chart-model.ts

```diff
@@ -16,31 +16,68 @@ export const BarChartModel = DotChartModel
   .named("BarChartModel")
   .props({
     type: typesPlotType("barChart"),
-    breakdownType: types.maybe(types.enumeration([...BreakdownTypes])),
+    breakdownType: types.optional(types.enumeration([...BreakdownTypes]), "count"),
     expression: types.maybe(Formula)
   })
+  .actions(self => ({
+    setBreakdownType(type: BreakdownType) {
+      self.breakdownType = type
```

Comment:
Should setting a breakdown type of `count` or `percent` clear the expression? If matching v2 behavior were the goal, the answer would be yes.